core/controllers/domain_objects_validator_test.py

```diff
@@ -245,3 +247,86 @@ def test_valid_images_do_not_raises_exception(self) -> None:
             ) as f:
                 files[filename] = f.read()
         domain_objects_validator.validate_suggestion_images(files)
+
+
+class ValidateExplorationChangeTests(test_utils.GenericTestBase):
```

Comment:
Re [this comment](https://github.com/oppia/oppia/pull/16065#discussion_r979391660):

> Also wanted to confirm that, the new changes are the one when you mentioned validation issues when converting to domain objects, right?

No, not quite. It looks like your function is still returning something but I'm not seeing checks that the returned value is correct, or that it detects errors within the sub-domain objects like WrittenTranslation.

Reply:
I see. Though I have a test which raises an error if we enter invalid values inside translations. Also added the test to check for the returned value. Please let me know if it sounds good.
Thanks